apps/docs/src/about/team.md

```diff
@@ -1,72 +1,128 @@
+---
+layout: page
+---
+
 <script lang="ts" setup>
-import { VPTeamMembers } from 'vitepress/theme'
+import { data } from "../index.data";
+import {
+  VPTeamPage,
+  VPTeamPageTitle,
+  VPTeamMembers,
+  VPTeamPageSection
+} from 'vitepress/theme'
+
+const shuffleArray = (array) =>
```

Comment:
I get your intention with shuffeling but imho this is confusing. What do you think about ordering the core team by title and the contributors by number/size of contributions